langchain4j-parent/pom.xml

```diff
@@ -479,6 +479,15 @@
     </distributionManagement>
 
     <profiles>
+        <profile>
+            <id>java11+</id>
+            <activation>
+                <jdk>[11,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
```

Comment:
I am not sure I understand the purpose of this, is it supposed to be 8 or 11?

Reply:
activation looks at the effective JDK you are building with
compiler.release defines the target release you are compiling to

then this profile reads: if you are compiling with a modern JDK (>11, that supports `javac --release`), use this release flag (that by default targets Java 8: this PR does not change anything at that level)

Reply:
@hboutemy I see, thanks for explanation!